src/modules/billing/jobs/prepare-transactions.js

```diff
@@ -59,9 +63,17 @@ const onComplete = async (job, queueManager) => {
 
     // If there's nothing to process, skip to cm refresh
     if (billingTransactionIds.length === 0) {
-      logger.info(`0 transactions for batch ${batchId}, requesting CM batch generation`);
-      await batchService.requestCMBatchGeneration(batchId);
-      await queueManager.add(refreshTotalsJobName, batchId);
+      logger.info(`No transactions left to process for batch ${batchId}. Requesting CM batch generation...`);
```

Comment:
I'm assuming this is regards the prod issue so I'm going to assume this is a-ok.